pipeline-manager: pipeline monitoring (#6096)
feldera/feldera: docs.feldera.com/docs/changelog.md

        ## Unreleased

>         Pipeline monitoring: Feldera now monitors each pipeline health and stores these as events

Reviewer: ```suggestion
        Pipeline monitoring: Feldera now monitors and persists a pipeline's health
```

>         in the database. They are exposed via `/v0/pipelines/[pipeline]/events` and further details

Reviewer: ```suggestion
        Events are queryable via `/v0/pipelines/[pipeline]/events` and further details
```

Author: Updated to:
```
        Pipeline monitoring: Feldera now monitors and persists a pipeline's health.
        Events are queryable via `/v0/pipelines/[pipeline]/events?selector=[all|status]`
        and a specific event via `/v0/pipelines/[pipeline]/events/[<event-id>|latest]?selector=[all|status]`.
```
